sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -358,7 +362,10 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
 
   // ManagementClient methods # Begin
 
-  async peekMessages(options: PeekMessagesOptions = {}): Promise<ReceivedMessage[]> {
+  async peekMessages(
+    maxMessageCount: number,
```

Comment:
Just checking, if the user passes undefined here (ie, doesn't specify the parameter in JS) will they get an appropriate error? It doesn't look like our invalidParameters.spec.ts covers that currently.

(I believe this would have been an issue before as well)

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/9798/commits/e6dd038f769835420110138b3bb64aa9018b4d2d